x/auth/ante/sigverify.go

```diff
@@ -250,6 +261,46 @@ func OnlyLegacyAminoSigners(sigData signing.SignatureData) bool {
 	}
 }
 
+func verifyIsOnCurve(pubKey cryptotypes.PubKey) (err error) {
```

Comment:
is there a performance overhead observed here?

Reply:
Didn't observe any noticable changes, do you want me to do some benchmark testing?